src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/Cas1AutoScript.kt

```diff
@@ -84,16 +97,42 @@ class Cas1AutoScript(
         ),
         documentation = "For local use in development and testing",
       ),
+      SeedUser(
+        username = "LAOFULLACCESS",
+        roles = listOf(
+          UserRole.CAS1_ASSESSOR,
+          UserRole.CAS1_MATCHER,
+          UserRole.CAS1_MANAGER,
+          UserRole.CAS1_WORKFLOW_MANAGER,
+          UserRole.CAS1_ADMIN,
+          UserRole.CAS1_REPORT_VIEWER,
+          UserRole.CAS1_APPEALS_MANAGER,
```

Comment:
I curious as to why you grant **all** the roles at this point? Are there scenarios where ADMIN and REPORT_VIEWER experience different behaviour around LAOs?